### ISSUE TITLE
Create subtask with Original Estimate

### ISSUE THREAD

**Issue by Blackbaud-KevinEnns** [Netflix-Skunkworks/go-jira#76]
Is there a way within the subtask flow to specify a value for the original estimate?

In the jira api, this appears to be under a timetracking field which contains both originalEstimate and remainingEstimate, like this:
"timetracking": {
            "originalEstimate": "10",
            "remainingEstimate": "5"
        }

I have tried a few things with overrides to see if I could get it to set, but it doesn't seem to work, presently.

**Comment by coryb** [comment]
This is on the create subtask api or the edit subtask api?  When the editor pops up you should be able to add something like:
```yaml
    timetracking:
      originalEstimate: 10
      remainingEstimate: 5
```
Which would be under a `fields` label in the YAML document.  I believe you could add the same content under for the edit template.  The document is directly translated into JSON and sent to the api service, so just make sure the template structure matches the expected JSON structure Jira expects.

If you want to make this generically available in your template you would create your own.  Here is some documentation on it: https://github.com/Netflix-Skunkworks/go-jira#templates

Basically you would export the right template (subtask probably) then add something like:
```yaml
    timetracking:
      originalEstimate: {{ or .overrides.originalEstimate "" }}
      remainingEstimate: {{ or .overrides.remainingEstimate ""}
```
Then you could run jira with `jira -t mytemplate subtask -o originalEstimate 10 -o remainingEstimate 5` to auto populate those fields, or just edit those fields which would default to the `""` values and fill in what you want in the structured document.

**Comment by Blackbaud-KevinEnns** [comment]
That works.  Thanks for the quick answer!

I'm really enjoying using this project.  Keep up the good work!
